qt/scientific_interfaces/ISISReflectometry/CMakeLists.txt

```diff
@@ -69,7 +69,9 @@ mtd_add_qt_library(
     MantidQtWidgetsCommon
     MantidQtIcons
   INSTALL_DIR_BASE ${PLUGINS_DIR}
-  OSX_INSTALL_RPATH @loader_path/../../Contents/MacOS
+  OSX_INSTALL_RPATH
+    @loader_path/../../Contents/MacOS
+    @loader_path/../../Contents/Frameworks
   LINUX_INSTALL_RPATH "\$ORIGIN/../../${LIB_DIR}"
 )
```

Comment:
Not related to the change, but had to add `${Boost_INCLUDE_DIRS}` to `INCLUDE_DIRS` to get it compile.

Reply:
Odd. I wonder why mine compiled. Was it just for this interface?

Reply:
So far yes, because I had added it to the most of the libraries some time ago.